src/aerie_cli/schemas/client.py

```diff
@@ -113,6 +113,7 @@ class EmptyActivityPlan(ClientSerialize):
     )
     end_time: Arrow = field(
         converter = arrow.get)
+    tags: List[Dict]
```

Comment:
Is there a reason this one isn't `Optional`?

Reply:
I don't think so- I can fix that now.